packages/bindgen/spec.yml

```diff
@@ -427,6 +427,11 @@ records:
       custom_http_headers: 
         type: 'std::map<std::string, std::string>'
         default: {}
+      client_validate_ssl:
+        type: bool
+        default: true
+      ssl_trust_certificate_path: util::Optional<std::string>
+      ssl_verify_callback: Nullable<SSLVerifyCallback>
```

Comment:
Should it be `off_thread`?

Reply:
It is. See line 642.

Reply:
Not sure if it can be marked as `off_thread` here.

As this is the `SSLVerifyCallback`:

```yml
SSLVerifyCallback:
  cppName: std::function<SyncConfig::SSLVerifyCallback>
```

And having `ssl_verify_callback` as:

```yml
SyncConfig:
  fields:
    # ...
    ssl_verify_callback: Nullable<SSLVerifyCallback>
```

It's unsure how/if it should be applied. These are 2 examples of other how other methods use it:

```yml
should_compact_on_launch_function: 'Nullable<std::function<(total_bytes: uint64_t, used_bytes: uint64_t) off_thread -> bool>>'
error_handler: 'Nullable<std::function<(session: SharedSyncSession, error: SyncError) off_thread -> void>>'
```

We do however need it for our helper `make_ssl_verify_callback`:

```yml
classes:
  Helpers:
    staticMethods:
      # ...
      make_ssl_verify_callback: '(callback: (server_address: const std::string&, server_port: int, pem_data: std::string_view, preverify_ok: int, depth: int) off_thread -> bool) -> SSLVerifyCallback'
```

But @RedBeard0531 could chime in here 🙁